packages/go_router/lib/src/route.dart

```diff
@@ -1317,13 +1406,26 @@ class StatefulNavigationShellState extends State<StatefulNavigationShell>
             key: ObjectKey(branch),
             branch: branch,
             navigatorForBranch: (StatefulShellBranch b) =>
```

Comment:
while you on it, can you also rename the parameter `b` to be something more descriptive. Our style guide suggest no abbreviation